packages/strapi-design-system/src/Select/SingleSelect.tsx

```diff
@@ -81,6 +80,7 @@ export interface SingleSelectInputProps extends Omit<SingleSelectPropsWithoutLab
   'aria-label'?: string;
   id?: string;
   label?: string;
+  onCloseAutoFocus?(event: Event): void;
```

Comment:
ℹ️ I'm not sure I've added this in the right spot and I don't know if we also want to add this for multi-select or not 🤔

Reply:
I would extend the type again doing `Pick<SelectParts.ContentProps, 'onCloseAutofocus'>` instead imo.

And for consistency so they're in alignment I would make the same change on the MultiSelect, the closer their APIs the easier it is to understand how to use them imo